### PR TITLE
Fixes #7198: Add bare ncf support to rtf

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7198

### CODE REVIEW

File: rtf [hunk 1]
```diff
@@ -102,8 +102,16 @@ def host_lines(platform, hosts, pf_id):
       elif host['rudder-setup'] != 'server':
         own_server = server
 
-      line = "configure(config, $" + os + ", '" + pf + "', " + str(pf_id) + ", '" + hostname +"', " + host_id
-      line += ", setup:'" + host['rudder-setup'] + "', version:'" +  host['rudder-version'] + "', server:'" + own_server + "', host_list:'" + hostnames + "'"
+      # generic host
+      line = "configure(config, $" + os + ", '" + pf + "', " + str(pf_id) + ", '" + hostname +"', " + host_id 
+      line += ", setup:'" + host['rudder-setup']  + "', host_list:'" + hostnames + "'"
+      # rudder host
+      if host['rudder-setup'] != 'ncf' and host['rudder-setup'] == 'empty':
```
Comment (VinceMacBuche) [inline rtf]: WHat would 'empty' rudder-setup mean ?

Comment (peckpeck) [inline rtf]: empty means just create a VM and do nothing.
This should be != and not ==

Comment (VinceMacBuche) [inline rtf]: Ok, then should be != as you said :D

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**